src/modules/csm/tree.py

```diff
@@ -0,0 +1,45 @@
+import json
+from dataclasses import dataclass
+from typing import Self, Sequence, TypeAlias
+
+from hexbytes import HexBytes
+from oz_merkle_tree import StandardMerkleTree
+
+from src.web3py.extensions.lido_validators import NodeOperatorId
+
+Leaf: TypeAlias = tuple[NodeOperatorId, int]
```

Comment:
```suggestion
Leaf: TypeAlias = tuple[NodeOperatorId, SharbI]
```

Reply:
From python3.12 you can do
```suggestion
type Leaf = tuple[NodeOperatorId, int]
```